include/dxc/Support/dxcapi.use.h

```diff
@@ -152,6 +180,26 @@ class DxcDllSupport {
   }
 };
 
+// This class is for instances where we *only* expect
+// to load dxcompiler.dll, and nothing else
+class DxCompilerDllLoader : public SpecificDllLoader {
+public:
+  HRESULT Initialize() {
+    return InitializeForDll(kDxCompilerLib, "DxcCreateInstance");
+  }
+};
+
+// This class is for instances where we want to load a
+// subset of any dlls that would give DxcLibrary functionality.
+// e.g, load a IDxcLibrary object.
+// Includes but isn't limited to dxcompiler.dll and dxil.dll
+class LibraryDllLoader : public SpecificDllLoader {
```

Comment:
Should this be called `DXCLibraryDllLoader`? "Library" on its own is a pretty generic word here ("library dynamically loaded library loader"?)